handlers/validator.go

```diff
@@ -626,10 +628,21 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Internal server error", http.StatusInternalServerError)
 		return
 	}
+
+	// remove scheduled committees
+	latestEpoch := services.LatestEpoch()
+	for _, syncPeriod := range syncPeriods {
+		if syncPeriod.FirstEpoch > latestEpoch {
+			syncPeriods = syncPeriods[1:]
+		} else {
+			break
+		}
+	}
```

Comment:
```
for i, syncPeriod := range syncPeriods {
	if syncPeriod.FirstEpoch <= latestEpoch {
		syncPeriods = syncPeriods[i:]
		break
	}
}
```

Reply:
maybe faster?

Reply:
See my comment in conversation above

Reply:
Duplicated conversation, we will resolve this one and remember it in the first one :)